pkg/controllers/raycluster_webhook_test.go

```diff
@@ -351,11 +366,25 @@ func TestValidateCreate(t *testing.T) {
 
 func TestValidateUpdate(t *testing.T) {
```

Comment:
It could be good to keep test case to validate update cluster without annotation and add annotation one as new test case.